src/Commands/DoctorCommand.php

```
@@ -100,24 +100,25 @@ private function checkInvoices(): void
-        $paidWithoutPdf = Invoice::where('status', InvoiceStatus::paid->value)
+        $paidWithoutPdf = Invoice::query()->where('status', InvoiceStatus::paid->value)
             ->whereNull('pdf_path')
             ->count();
 
         if ($paidWithoutPdf > 0) {
             $this->warn("  ⚠️  {$paidWithoutPdf} paid invoices are missing PDFs");
 
             // Show sample
-            $sample = Invoice::where('status', InvoiceStatus::paid->value)
+            /** @var Invoice|null $sample */
+            $sample = Invoice::query()->where('status', InvoiceStatus::paid->value)
```

PHPStan (2.1.37)

[error] Call to an undefined method Illuminate\Database\Query\Builder::with().


(method.notFound)